src/EnergyPlus/RefrigeratedCase.cc

```diff
@@ -7867,62 +7866,62 @@ void SetupReportInput(EnergyPlusData &state)
                 // For "IDOut" variable in SetupOutputVariable, need to create a single name that includes
                 //    both the walk-in name and the zone name - see "Walkin_and_zone_name" concatination
                 //    This new variable name is important if using an rvi file!
-                for (int zoneId = 1; zoneId <= WalkIn(walkInNum).NumZones; ++zoneId) {
+                for (int zoneId = 1; zoneId <= walkin.NumZones; ++zoneId) {
 
-                    Walkin_and_zone_name = WalkIn(walkInNum).Name + "InZone" + WalkIn(walkInNum).ZoneName(zoneId);
+                    Walkin_and_zone_name = walkin.Name + "InZone" + walkin.ZoneName(zoneId);
```

Comment:
`format` here.

Reply:
Ahh, I wonder how that slipped through the auto-format pass that @jmarrec did a while back.  Maybe we should somehow delete the `std::string operator+` function lol. I'm kidding!